tests/conftest.py

```diff
@@ -1255,3 +1277,14 @@ class UseGanache:
     @pytest.fixture(autouse=True)
     def _start_ganache(self, ganache):
         """Start a Ganache image."""
+
+
+@pytest.fixture()
+def change_directory():
```

Comment:
we have this functionality in helpers base

Reply:
I guess you mean `aea.helpers.cd`, but it is not a pytest fixture (in fact I use it in this fixture)